### PR TITLE
Add a combined project sample for Teams tab and Outlook add-in

### PR DESCRIPTION
**Pull request by 1yefuwang1**

### CODE REVIEW

File: teams-tab-with-outlook-addin/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Publish to Teams
+
+Once deployed, you may want to distribute your application to your organization's internal app store in Teams. Your app will be submitted for admin approval.
+
+- From Visual Studio Code: open the Teams Toolkit and click `Publish to Teams` or open the command palette and select: `Teams: Publish to Teams`.
+- From TeamsFx CLI: run command `teamsfx publish` in your project directory.
+
+## Add Single Sign On feature
+
```
Comment (MuyangAmigo) [inline teams-tab-with-outlook-addin/README.md:84]: This section is not relevant, we should remove it.

File: teams-tab-with-outlook-addin/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- From Visual Studio Code: open the command palette and select: `Teams: Validate manifest file`.
+- From TeamsFx CLI: run command `teamsfx validate` in your project directory.
+
+## Package
+
+- From Visual Studio Code: open the Teams Toolkit and click `Zip Teams app package` or open the command palette and select `Teams: Zip Teams app package`.
+- Alternatively, from the command line run `teamsfx package` in the project directory.
+
+## Publish to Teams
+
```
Comment (MuyangAmigo) [inline teams-tab-with-outlook-addin/README.md:77]: This section is also not relevant, can we remove it?

File: teams-tab-with-outlook-addin/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+| <ul><li>Open Teams Toolkit, and sign into Azure by clicking the `Sign in to Azure` under the `ACCOUNTS` section from sidebar.</li> <li>After you signed in, select a subscription under your account.</li><li>Open the Teams Toolkit and click `Provision in the cloud` from DEVELOPMENT section or open the command palette and select: `Teams: Provision in the cloud`.</li><li>Open the Teams Toolkit and click `Deploy to the cloud` or open the command palette and select: `Teams: Deploy to the cloud`.</li></ul> | <ul> <li>Run command `teamsfx account login azure`.</li> <li>Run command `teamsfx account set --subscription <your-subscription-id>`.</li> <li> Run command `teamsfx provision`.</li> <li>Run command: `teamsfx deploy`. </li></ul> |
+
+> Note: Provisioning and deployment may incur charges to your Azure Subscription.
+
+## Preview
+
```
Comment (MuyangAmigo) [inline teams-tab-with-outlook-addin/README.md:53]: Does remote preview work this way?

File: teams-tab-with-outlook-addin/README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+- Use a different tunneling solution or customize the defaults
+- Add custom steps to debugging, provisioning, deploying, publishing, etc.
+
+### What about my existing Teams Toolkit projects?
+The changes in this pre-release require upgrades to the TeamsFx configuration files. We recommend that you create a new app using this version. In the future, we'll provide a way to automatically upgrade existing Teams apps that were created with a previous version of Teams Toolkit.
+
+Learn more about the changes in this pre-release at [https://aka.ms/teamsfx-v5.0-guide](https://aka.ms/teamsfx-v5.0-guide).
+
+# How to use this Teams Tab app HelloWorld app
+
```
Comment (MuyangAmigo) [inline teams-tab-with-outlook-addin/README.md:20]: How to use this Hello World Teams tab and Outlook Add-in sample app

File: teams-tab-with-outlook-addin/README.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.
+
```
Comment (MuyangAmigo) [inline teams-tab-with-outlook-addin/README.md:22]: Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.

Outlook add-ins are integrations built by third parties into Outlook by using our web-based platform. 

Now you have the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Teams manifest.

File: hello-world-teams-tab-and-outlook-addin/README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+Now you have the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Teams manifest.
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/): version 16 or 18.
+- Edge or Chrome installed.
+- Outlook for debugging Outlook add-in.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 4.0.0 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
+
```
Comment (MuyangAmigo) [inline hello-world-teams-tab-and-outlook-addin/README.md:18]: This is not correct, it should be the Teams Toolkit Prerelease version.

File: hello-world-teams-tab-and-outlook-addin/README.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+Outlook add-ins are integrations built by third parties into Outlook by using our web-based platform.
+
+Now you have the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Teams manifest.
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/): version 16 or 18.
+- Edge or Chrome installed.
+- Outlook for debugging Outlook add-in.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
```
Comment (MuyangAmigo) [inline hello-world-teams-tab-and-outlook-addin/README.md:16]: It should be Outlook Desktop Client on Windows?

File: hello-world-teams-tab-and-outlook-addin/README.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+- From TeamsFx CLI: Start debugging the project by executing the command `teamsfx preview --env local` in your project directory.
+
+## Debug Outlook add-in
+- From Visual Studio Code only: use the `Run and Debug Activity Panel` in Visual Studio Code, select `Outlook Desktop(Edge Chromium)`, and click the `Run and Debug` green arrow button.
+
+## Edit the manifest
+
+You can find the Teams app manifest in `./appPackage` folder. The folder contains one manifest file:
+* `manifest.template.json`: Manifest file for Teams app running locally or running remotely (After deployed to Azure).
+
```
Comment (MuyangAmigo) [inline hello-world-teams-tab-and-outlook-addin/README.md:32]: It should be `manifest.json`. and Remove the mention of "Teams app", just "app" is fine.

File: hello-world-teams-tab-and-outlook-addin/README.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+| <ul><li>Open Teams Toolkit, and sign into Azure by clicking the `Sign in to Azure` under the `ACCOUNTS` section from sidebar.</li> <li>After you signed in, select a subscription under your account.</li><li>Open the Teams Toolkit and click `Provision in the cloud` from DEVELOPMENT section or open the command palette and select: `Teams: Provision in the cloud`.</li><li>Open the Teams Toolkit and click `Deploy to the cloud` or open the command palette and select: `Teams: Deploy to the cloud`.</li></ul> | <ul> <li>Run command `teamsfx account login azure`.</li> <li>Run command `teamsfx account set --subscription <your-subscription-id>`.</li> <li> Run command `teamsfx provision`.</li> <li>Run command: `teamsfx deploy`. </li></ul> |
+
+> Note: Provisioning and deployment may incur charges to your Azure Subscription.
+
+## Preview Teams Tab
+
```
Comment (MuyangAmigo) [inline hello-world-teams-tab-and-outlook-addin/README.md:46]: Title can change to Preview the app? Or does Teams / Outlook portion previews differently?

**Review by MuyangAmigo — CHANGES_REQUESTED** [review]

File: hello-world-teams-tab-and-outlook-addin/README.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+# How to use this HelloWorld Teams tab and Outlook Add-in sample app
+
+Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.
+
+Outlook add-ins are integrations built by third parties into Outlook by using our web-based platform.
+
+Now you have the ability to create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema, based on the current JSON-formatted Teams manifest.
+
+## Prerequisites
+
```
Comment (MuyangAmigo) [inline hello-world-teams-tab-and-outlook-addin/README.md:12]: Can we also add a note here to remind developers that they need to use the same account in Teams Toolkit as logged in to Outlook

File: hello-world-teams-tab-and-outlook-addin/README.md (hunk 9 above)
Comment (1yefuwang1) [reply hello-world-teams-tab-and-outlook-addin/README.md:46]: Preview here means launch remote, which is not supported with one click F5 for Outlook.

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**